lib/send_grid_mailer/dev_deliverer.rb

```diff
@@ -24,7 +25,13 @@ def api_key
     end
 
     def letter_opener_delivery_method
-      @letter_opener_delivery_method ||= LetterOpener::DeliveryMethod.new(location: '/tmp/mails')
+      @letter_opener_delivery_method ||= LetterOpener::DeliveryMethod.new(location: dev_emails_location)
+    end
+
+    def dev_emails_location
+      Rails.application.config.action_mailer.sendgrid_dev_settings[:emails_location] || "/tmp/mails"
```

Comment:
Could you please document this in the `sendgrid_dev_settings` part of the readme, mentioning it as an optional configuration?